scopes/component/ui/component-meta/component-overview.tsx

```diff
@@ -40,13 +40,13 @@ export function ComponentOverview({
   ...rest
 }: ComponentOverviewProps) {
   const componentId = component?.id;
-  const tabsComponentId: Content[] = [
+  const tabsComponentId: ContentTab[] = [
     {
-      title: 'Component ID',
+      component: () => <span>Component ID</span>,
```

Comment:
Add this
`/* eslint-disable react/display-name */`
to the top of the file to avoid the eslint error.